### PR TITLE
Task/fix order of css attributes

### PR DESCRIPTION
**Pull request by lightblader**

### CODE REVIEW

File: scss/custom/_modal.scss [hunk 1]
```diff
@@ -22,8 +22,8 @@
   .modal-header,
   .modal-footer {
     // Fixed for older phones
-    display: flex;
     z-index: 1;
+    display: flex;
```
Comment (BastianHofmann) [inline scss/custom/_modal.scss:26]: @lightblader Could you explain why this change is required?

Comment (lightblader) [reply scss/custom/_modal.scss:26]: Because due to linter errors the version bump command cannot be executed

**Review by BastianHofmann — APPROVED** [review]
I don't get why the linter would complain about this, but ok.
